Kickstarter-iOS/Views/Cells/DeprecatedRewardCell.swift

```diff
@@ -1,270 +0,0 @@
-import KsApi
-import Library
-import Prelude
-
-internal protocol DeprecatedRewardCellDelegate: AnyObject {
```

Comment:
I noticed that `Nib.swift` still has a `case DeprecatedRewardCell`. Could you remove it?